### ISSUE TITLE
Show roman-chord conversion in Colab Notebook

### ISSUE THREAD

**Issue by titus-ong** [titus-ong/chordparser#3]
Show example of converting chords to roman numerals and back (preferably do this after #1).

**Comment by titus-ong** [comment]
To be included in both colab notebook and readthedocs.

**Comment by titus-ong** [comment]
Strictly using this issue for Colab Notebook (which has been completed). Will open a new issue for the docs on readthedocs.
